QualityControl/public/Model.js

```diff
@@ -87,6 +88,8 @@ export default class Model extends Observable {
     this.ws.addListener('close', this.handleWSClose.bind(this));
 
     this.initModel();
+
+    this.services.filter.initFilterService();
```

Comment:
I believe the services are currently initiliazed in handleLocationChange. It would be good if you initialize that one there as well https://github.com/AliceO2Group/WebUi/pull/2691/files#diff-8829298f0249abbf3900a4d93c5d0d84fc5bb4e38bfcd19be06a9d0b42657522R177